pkg/model/manifest.go

```diff
@@ -98,11 +98,15 @@ type Manifest struct {
 	Build         ManifestBuild           `json:"build,omitempty" yaml:"build,omitempty"`
 	Dependencies  ManifestDependencies    `json:"dependencies,omitempty" yaml:"dependencies,omitempty"`
 	GlobalForward []forward.GlobalForward `json:"forward,omitempty" yaml:"forward,omitempty"`
+	DevWarnings   DevWarnings
 
 	Type     Archetype `json:"-" yaml:"-"`
 	Manifest []byte    `json:"-" yaml:"-"`
 	IsV2     bool      `json:"-" yaml:"-"`
 }
+type DevWarnings struct {
```

Comment:
On compose we need to store this because the warnings are shown after the injecting ports from images, but since we only have to show the users which services has been sanitised I'd moved the `map[string]string` to the function itself